app/validators/form_password_validator.rb

```diff
@@ -63,26 +48,4 @@ def passwords_match
       )
     end
   end
-
-  def min_password_score
-    IdentityConfig.store.min_password_score
```

Comment:
we can remove this config value too now right? from `identity_config.rb` and `application.yml.default`